src/graphql/admin/queries.ts

```diff
@@ -7,22 +9,32 @@ import TransactionByIdQuery from "./root/query/transaction-by-id"
 import TransactionsByHashQuery from "./root/query/transactions-by-hash"
 import AccountDetailsByUserPhoneQuery from "./root/query/account-details-by-phone"
 import AccountDetailsByUsernameQuery from "./root/query/account-details-by-username"
+import AccountsDetailsByCustomFieldQuery from "./root/query/accounts-details-by-custom-field"
 import ListWalletIdsQuery from "./root/query/all-walletids"
 import WalletQuery from "./root/query/wallet"
 
+const fields = {
+  allLevels: AllLevelsQuery,
+  accountDetailsByUserPhone: AccountDetailsByUserPhoneQuery,
+  accountDetailsByUsername: AccountDetailsByUsernameQuery,
+  transactionById: TransactionByIdQuery,
+  transactionsByHash: TransactionsByHashQuery,
+  lightningInvoice: LightningInvoiceQuery,
+  lightningPayment: LightningPaymentQuery,
+  listWalletIds: ListWalletIdsQuery,
+  wallet: WalletQuery,
+}
+
+const { customFields } = getAccountsConfig()
+if (customFields && customFields.length > 0) {
+  Object.assign(fields, {
+    accountsDetailsByCustomField: AccountsDetailsByCustomFieldQuery,
+  })
+}
+
 const QueryType = GT.Object({
   name: "Query",
-  fields: () => ({
-    allLevels: AllLevelsQuery,
-    accountDetailsByUserPhone: AccountDetailsByUserPhoneQuery,
-    accountDetailsByUsername: AccountDetailsByUsernameQuery,
-    transactionById: TransactionByIdQuery,
-    transactionsByHash: TransactionsByHashQuery,
-    lightningInvoice: LightningInvoiceQuery,
-    lightningPayment: LightningPaymentQuery,
-    listWalletIds: ListWalletIdsQuery,
-    wallet: WalletQuery,
-  }),
+  fields: () => fields,
```

Comment:
no adding tracing here? why is there an asymmetry with the mutation?

Reply:
Fixed https://github.com/GaloyMoney/galoy/pull/1465/commits/969d2247d4e9da3054663dd7a280800526daa6cd